YAXLib/Custom/CustomSerializerWrapper.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
+// Licensed under the MIT license.
+
+using System;
+using System.Reflection;
+using System.Xml.Linq;
+
+namespace YAXLib;
+
+/// <summary>
+/// A wrapper around an <see cref="ICustomSerializer{T}"/>.
+/// It's methods are invoked with <see cref="System.Reflection"/>.
+/// </summary>
+internal class CustomSerializerWrapper : ICustomSerializer<object>
```

Comment:
Use a wrapper for the `ICustomSerializer` instances.
The wrapper includes Reflection code to invoke `ICustomSerializer` methods.